<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>455</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Monday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thursday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Friday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saturday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sunday</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Day One</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Day Two</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Monday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thursday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Friday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saturday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sunday</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>711</width>
      <height>191</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>711</width>
      <height>0</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>User Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Day One Times</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Day Two Times</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Availability</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Day One Slot</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Day Two Slot</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
